assets/js/neoforge.js

```diff
@@ -9,14 +9,26 @@ async function loadLatestVersions(minecraftVersions) {
 	let gav;
 	let fn;
 	let mcvers;
+    let note;
+    let dropDown_VAL;
+    let badges_one;
+    let badges_two;
 	if (mcVersion.startsWith("1.20.1")) {
 		gav = LEGACY_GAV;
-                fn = "forge";
-		mcvers = "1.20.1"
+        fn = "forge";
+		mcvers = "1.20.1";
+        note = `<b>Note:</b> This file is still called <i>forge</i> because we're trying to maintain compatibility with launchers, assuming they don't hardcode things too much.`;
+        dropDown_VAL = ` open="open"`;
+        badges_one = `<font class="badges_stable">RECOMMENDED</font>`;
```

Comment:
I don't know of a good reason why 1.20.1 would be marked as "recommended". We don't recommend 1.20.1 over any other version.

Reply:
Because 1.20.2 is still in beta. OK, I will change recommend with Stable.

Reply:
I would prefer not having a badge for 1.20.1 that remarks it as either recommended or stable. There's nothing particularly important about 1.20.1 which merits a badge.

On the other hand, NF 20.1 merits a `BETA` badge because it is in beta.